app/src/main/java/org/mukdongjeil/mjchurch/util/MyFirebaseMessagingService.java

```diff
@@ -0,0 +1,61 @@
+package org.mukdongjeil.mjchurch.util;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.util.Log;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import org.mukdongjeil.mjchurch.R;
+import org.mukdongjeil.mjchurch.ui.MainActivity;
+
+import androidx.core.app.NotificationCompat;
+
+public class MyFirebaseMessagingService extends FirebaseMessagingService {
+    private static final String TAG = MyFirebaseMessagingService.class.getSimpleName();
+
+    @Override
+    public void onMessageReceived(RemoteMessage remoteMessage) {
+        super.onMessageReceived(remoteMessage);
+        if (remoteMessage.getNotification() != null) {
+            Log.d(TAG, "Message Notification Body : " + remoteMessage.getNotification().getBody());
+            displayNotificationOnSystemBar(remoteMessage.getNotification());
+        }
+    }
+
+    private void displayNotificationOnSystemBar(RemoteMessage.Notification notification) {
+        String title = notification.getTitle();
+        String message = notification.getBody();
+
+        Intent intent = new Intent(this, MainActivity.class);
+        intent.putExtra("remoteMessage", message);
+        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TOP);
+        PendingIntent pendingIntent = PendingIntent.getActivity(this,0, intent, PendingIntent.FLAG_ONE_SHOT);
+
+        String defaultChannelId = getString(R.string.default_notification_channel_id);
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        NotificationCompat.Builder notificationBuilder;
+
+        NotificationChannel channel;
+
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+            channel = new NotificationChannel(defaultChannelId, "name", NotificationManager.IMPORTANCE_HIGH);
```

Comment:
how about change that string to defaultChannelId?
```suggestion
            channel = new NotificationChannel(defaultChannelId, defaultChannelId, NotificationManager.IMPORTANCE_HIGH);
```